formver.compiler-plugin/core/src/org/jetbrains/kotlin/formver/core/embeddings/expression/debug/Utils.kt

```diff
@@ -33,6 +33,7 @@ val PermExp.debugTreeView: TreeView
     get() = when (this) {
         is PermExp.WildcardPerm -> PlaintextLeaf("wildcard")
         is PermExp.FullPerm -> PlaintextLeaf("write")
+        is PermExp.EpsilonPerm -> PlaintextLeaf("read")
```

Comment:
I don't think this is needed, wildcard does the same thing.

Reply:
I see, makes sense. I just wasn't sure how exactly wildcard permissions worked which is why I added the epsilon permission, but I removed it now